services/QuillLMS/client/app/bundles/Staff/components/evidence/llmPromptDatasets/individualTrial/__tests__/viewLLMPromptModal.test.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import ViewLLMPromptModal from '../viewLLMPromptModal';
+import { LLMPromptInterface } from '../../../../../interfaces/evidenceInterfaces';
+
+describe('ViewLLMPromptModal', () => {
+  const mockLLMPrompt: LLMPromptInterface = {
+    id: 1,
+    name: 'Sample Prompt',
+    prompt: 'This is the prompt content for testing.',
+    locked: false,
+    optimal_examples_count: 3,
+    optimal_guidelines_count: 2,
+    suboptimal_examples_count: 1,
+    suboptimal_guidelines_count: 1,
+    guidelines_count: 5,
+    created_at: '2024-11-07T00:00:00Z',
+    updated_at: '2024-11-07T00:00:00Z',
+    llm_prompt_template_id: 101,
+  };
+
+  const mockCloseModal = jest.fn();
+
+  const defaultProps = {
+    llmPrompt: mockLLMPrompt,
+    closeModal: mockCloseModal
+  }
+
+  it('matches the snapshot with sample LLM prompt data', () => {
+    const { asFragment } = render(
+      <ViewLLMPromptModal {...defaultProps} />
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('displays the correct LLM Prompt information', () => {
+    render(<ViewLLMPromptModal {...defaultProps} />);
+
+    expect(screen.getByText('LLM Prompt Template')).toBeInTheDocument();
+    expect(screen.getByText('Sample Prompt')).toBeInTheDocument();
+    expect(screen.getByText('Contents')).toBeInTheDocument();
+    expect(screen.getByText('This is the prompt content for testing.')).toBeInTheDocument();
```

Comment:
If you do decide to update to `const` declarations for these comparisons, just a quick flag that you'd want one for `Sample Prompt` and `This is the prompt content for testing.`.  Which would also make it a bit more explicit that the `LLM Prompt Template` and `Contents` text are not values that we expect because of testing setup, but are expected because of the root component itself.